src/stores/entries.js

```diff
@@ -263,6 +263,7 @@ export const useEntryStore = defineStore('entries', {
         await errorStore.throwError(error, 'Error deleting entry')
       }
       this._isLoading = false
+      this._entries = this._entries.filter((entry) => entry.id !== entryId)
```

Comment:
Maybe move this line after Promise.all and before the catch block. Because in case an error occurs, the catch block will execute and after that this line will also be executed, even if the entry has not been removed from the db.

Reply:
Yes, you are right, it definitely should be inside try catch block, missed it.